lib/ui/channel_buffers.dart

```diff
@@ -380,7 +380,7 @@ class ChannelBuffers {
     assert(!name.contains('\u0000'), 'Channel names must not contain U+0000 NULL characters.');
     final _Channel channel = _channels.putIfAbsent(name, () => _Channel());
     channel.setListener(callback);
-    channelListenedTo(name, true);
+    sendChannelUpdate(name, true);
```

Comment:
I'm not sure what our style guide is here, but I'd consider making this a named argument for clarity:

```suggestion
    sendChannelUpdate(name, listening: true);
```